<a href="https://colab.research.google.com/github/annanya-mathur/Building-my-own-Artificial-Neural-Network-with-Tensorflow/blob/main/ANN_keras.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **This neural network is made with 1 input and output layer and 2 hidden layers with keras as api and tensorflow at backend**

In [1]:
import keras

In [3]:
from keras.models import Sequential
from keras.layers import Dense
from keras.optimizers import Adam

In [5]:
model=Sequential(
    [
     Dense(16,input_dim=1,activation="relu"),
     Dense(36,activation="relu"),
     Dense(2,activation="softmax")
    ]
)

In [11]:
model.compile( Adam(learning_rate=0.001),
               loss="sparse_categorical_entropy",
              metrics=["accuracy"]
)

In [14]:
model.summary()

Model: "sequential"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
dense (Dense)                (None, 16)                32        
_________________________________________________________________
dense_1 (Dense)              (None, 36)                612       
_________________________________________________________________
dense_2 (Dense)              (None, 2)                 74        
Total params: 718
Trainable params: 718
Non-trainable params: 0
_________________________________________________________________


>

**1.   Hidden layer 1 has 32 parameters**

**2.   Hidden layer 2 has 612 parameters**

